tower-balance/src/lib.rs

```diff
@@ -106,6 +106,22 @@ where
         }
     }
 
+    /// Returns true iff there are ready services.
+    pub fn is_empty(&self) -> bool {
```

Comment:
Can this be named `is_ready(&self)`? Does this function returning `false` imply that `poll_ready` will return `Ready`?